perf/bumbleBench/build.xml

```diff
@@ -19,16 +19,32 @@
 	<description>BumbleBench Suite</description>
 
 	<!-- Set global properties for this build -->
+	<property environment="env" />
 	<property name="DEST" value="${BUILD_ROOT}/perf/bumbleBench" />
 	<property name="SRC" location="." />
 	
+	<condition property="isZOS" value="true">
+		<equals arg1="${os.name}" arg2="z/OS"/>
+	</condition>
+		
+	<condition property="git-prefix" value="git" else="https">
+		<isset property="isZOS"/>
+	</condition>
+		
+	<condition property="bumblebench_repo" value="${env.BUMBLEBENCH_REPO}" else="${git-prefix}://github.com/AdoptOpenJDK/bumblebench.git">
+		<isset property="env.BUMBLEBENCH_REPO"/>
+	</condition>
+	<condition property="bumblebench_branch" value="${env.BUMBLEBENCH_BRANCH}" else="master">
+		<isset property="env.BUMBLEBENCH_BRANCH"/>
+	</condition>	
```

Comment:
Rather than referencing BUMBLEBENCH_REPO/BRANCH env vars, just set to master.  This file can be changed in a developers openjdk-tests repo/branch to point to their bumblebench branch in the cases where its needed.